TTS/trainer.py

```diff
@@ -115,6 +117,9 @@ def __init__(
             tb_logger (TensorboardLogger, optional): Tensorboard logger. If not provided, the default logger is used.
                 Defaults to None.
 
+            wandb_logger (WandbLogger, optional): W&B logger. If not provided, the default logger is used.
```

Comment:
As below for `process_args`, I think wanddb_logger needs to replace tb_logger if it is the choosen `log_dashboard`. I guess it requires wandb_logger to implement the same function names as tb_logger

Reply:
Okay, I'll try to do it. We might need one extra condition to flush the metrics after every epoch for wandb_logger. Other than that I think all the things can basically remain the same.